pkg/metaclient/kv.go

```diff
@@ -0,0 +1,47 @@
+package metaclient
+
+import "context"
+
+type Txn interface {
+	// Cache Ops in the Txn
+	Do(ops ...Op) Txn
+
+	// Commit tries to commit the transaction.
+	// Any Op error will cause entire txn rollback and return error
+	Commit(ctx context.Context) (*TxnResponse, error)
+}
+
+type KV interface {
+	// Put puts a key-value pair into metastore.
+	// Note that key,value can be plain bytes array and string is
+	// an immutable representation of that bytes array.
+	// To get a string of bytes, do string([]byte{0x10, 0x20}).
+	// When passed WithTTL(ttl), Put will insert a key with ttl duration.
```

Comment:
```suggestion
	// When WithTTL(ttl) is passed, Put will insert a key with ttl duration.
```
Please fix grammar in similar sentences.

Reply:
Get it.